vNext/src/OrleansAWSUtils/OrleansAWSUtils.csproj

```diff
@@ -3,9 +3,7 @@
     <IsPackable>true</IsPackable>
     <PackageId>Microsoft.Orleans.OrleansAWSUtils</PackageId>
     <Title>Microsoft Orleans AWS Utilities</Title>
-    <Description>
-      Library of utility types for Amazon AWS of Microsoft Orleans.
-    </Description>
+    <Description>Library of utility types for Amazon AWS of Microsoft Orleans.</Description>
```

Comment:
Not your fault, but if you're okay making revisions to this, could we write:
`Support library for hosting Orleans on Amazon AWS`

That way it matches the description for Azure utils